janus_core/calculations/md.py

```diff
@@ -607,7 +617,12 @@ def _run_dynamics(self) -> None:
             if self.logger:
                 self.logger.info("Beginning temperature ramp at %sK", temps[0])
             for temp in temps:
+                if temp == 0.0:
```

Comment:
is intentional.. i want to catch when the user puts 0.0 in the input... although you may be right since there is a np.arange that provides 0.0 now and not the user.